Libs/GObject/Classes/Object.cs

```diff
@@ -76,7 +77,7 @@ public Object(params ConstructParameter[] properties)
                     values[i] = prop.Value;
                 }
 
-                // Create with properties
+                // Create with propeties
```

Comment:
Typo regression

Reply:
Funnily enough, I just pushed a commit that fixed this about ten seconds ago.